hyppo/kgof/density.py

```diff
@@ -12,8 +10,8 @@
 from abc import ABC, abstractmethod
 import autograd
 import autograd.numpy as np
-from .data import DSNormal, DSIsotropicNormal
-import scipy.stats as stats
+from .datasource import DSNormal, DSIsotropicNormal
+import scipy.stats as sp
```

Comment:
what was wrong with stats?

Reply:
Probably was just in a hurry when I was editing the imports, no reason to change it!